### ISSUE TITLE
AoE form should prevent screen readers from accessing content outside of modal

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4053]
## Area or Component
Conduct Test

## Quick Summary
Screen readers can read content outside the modal dialog.

## Accessibility Violation
* `reading-order-browse-outside-modal`
    * Screen readers can read content outside the modal dialog.

## Action Requested
* Modal should prevent screen readers from accessing content from outside modal
 * Dev approach TBD - per Alis - To hide an element to screen readers (and child elements), simply add the aria-hidden="true" attribute.

## Original Deque Issue
* https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal

**Comment by mehansen** [comment]
paired with @emmastephenson using the axe Chrome plugin to try to understand what element is triggering this violation but the tool doesn't give any indication

the only elements not tagged with `aria-hidden="true"` when the modal is open are empty `div` elements and some `<script>` and `<noscript>` elements ([which don't use aria attributes](https://www.w3.org/TR/html-aria/#el-script)). reached out to Deque and they think it might be the `div` elements that are triggering the violation, but these are added by the [react-modal](https://github.com/reactjs/react-modal) component to trap focus per https://github.com/reactjs/react-modal/issues/742

I'm unable to reproduce tabbing focus out of the modal and @nathancrtr and I also tested with a screenreader and weren't able to read anything outside of the modal so I think this is a false positive

will close this ticket and any others related to `reading-order-browse-outside-modal` for modals using react-modal if I don't get a helpful response from Deque within a couple days

**Comment by mehansen** [comment]
haven't heard back from Deque, so closing per my earlier comment
